portfolio/src/main/java/com/google/sps/Data/Quotes.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+import java.util.List;
+
+/** Class containing quotes 
+* which consists of a name/title and it's corresponding list 
+* of quotes.
+*/
+public class Quotes {
+
+    private String name;
+    private List<String> quotes;
```

Comment:
Having quotes variable name inside Quotes class doesn't sound right. Maybe we can rename one of these accordingly (use messages as the class variable name)?

Reply:
Having a different thought, can we not make this class name as quote, and use List<Quote> in GetQuotesResponse? That means, each object in the quote class will have one quote. That might seem to be a cleaner design?

Reply:
Each object with a quote class having one quote will create a problem. As I am making a JSON file of quotes which is having a title and it's corresponding quotes. GetQuotesResponse already contains a List<Quotes>. Actually I was just trying to publish quotes one by one. But in the actual comment section all comments will be displayed on the main page. So then this will work fine accordingly. That's why I kept it as a JSON file of name/title and it's corresponding all quotes/messages. Will add the comment section in the next pull request.

Reply:
Ok sure. I was mostly trying to comment from the design perspective of things.

Reply:
Yes, I will keep this in mind for the comment section. To make it simpler and understandable.